packages/react-instantsearch-hooks/README.md

```diff
@@ -468,6 +468,132 @@ function Hits(props) {
 }
 ```
 
+### `useMenu`
+
+> `(props: UseMenuProps) => MenuRenderState`
+
+Hook to use a [menu](https://www.algolia.com/doc/api-reference/widgets/menu/js/).
+
+**Types**
+
+<details>
+<summary><code>UseMenuProps</code></summary>
+
+```ts
+type UseMenuProps = {
+  /**
+   * The name of the attribute in the records.
+   */
+  attribute: string;
+  /**
+   * The max number of items to display when
+   * `showMoreLimit` is not set or if the widget is showing less value.
+   */
+  limit?: number;
+  /**
+   * Whether to display a button that expands the number of items.
+   */
+  showMore?: boolean;
+  /**
+   * The max number of items to display if the widget
+   * is showing more items.
+   */
+  showMoreLimit?: number;
+  /**
+   * How to sort refinements. Possible values: `count|isRefined|name:asc|name:desc`.
+   *
+   * You can also use a sort function that behaves like the standard Javascript [compareFunction](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Syntax).
+   *
+   * If a facetOrdering is set in the index settings, it is used when sortBy isn't passed
+   */
+  sortBy?: SortBy<MenuItem>;
+  /**
+   * Function to transform the items passed to the templates.
+   */
+  transformItems?: TransformItems<MenuItem>;
+};
+```
+
+</details>
+
+<details>
+<summary><code>MenuRenderState</code></summary>
+
+```ts
+export type MenuItem = {
+  /**
+   * The value of the refinement list item.
+   */
+  value: string;
+  /**
+   * Human-readable value of the refinement list item.
+   */
+  label: string;
+  /**
+   * Number of matched results after refinement is applied.
+   */
+  count: number;
+  /**
+   * Indicates if the list item is refined.
+   */
+  isRefined: boolean;
+};
+
+type MenuRenderState = {
+  /**
+   * The list of filtering values returned from Algolia API.
+   */
+  items: MenuItem[];
+  /**
+   * Creates the next state url for a selected refinement.
+   */
+  createURL: (value: string) => string;
+  /**
+   * Action to apply selected refinements.
+   */
+  refine(value: string): void;
+  /**
+   * Send event to insights middleware
+   */
+  sendEvent: (
+    eventType: string,
+    facetValue: string,
+    eventName?: string
+  ) => void;
+  /**
+   * `true` if a refinement can be applied.
+   */
+  canRefine: boolean;
+  /**
+   * `true` if the toggleShowMore button can be activated (enough items to display more or
+   * already displaying more than `limit` items)
+   */
+  canToggleShowMore: boolean;
+  /**
+   * True if the menu is displaying all the menu items.
+   */
+  isShowingMore: boolean;
+  /**
+   * Toggles the number of values displayed between `limit` and `showMoreLimit`.
+   */
+  toggleShowMore: () => void;
+};
+```
+
+</details>
+
+**Example**
+
+```jsx
+function Menu(props) {
+  const { items } = useMenu(props);
```

Comment:
I would show all the parameters here, not just items.
I find that makes the example easier to copy-paste if not working with typescript.

Reply:
can do that after on all already made examples?